centos-ci/utils/get-vm-ip

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+
+import sys
+import time
+import libvirt
```

Comment:
The bindings are installed yes. This is the same as `virsh domifaddr` minus parsing output made for humans. This was a bit of an excuse for me to try it out so that I can revamp my `domssh` and `domip`  [scripts](https://github.com/jlebon/files/blob/master/bin/domssh).